amgut/handlers/auth_handlers.py

```diff
@@ -42,7 +42,7 @@ def post(self):
             ag_login_id, skid, kitinfo['password'],
```

Comment:
this is tangential to the PR, but, possible to have the password get encrypted if necessary? see here for how to do it:

https://github.com/biocore/american-gut-web/blob/master/scripts/ag#L104

Not _ideal_, but it would work to add the kit then, if `kitinfo['password']` does not identify, issue the method call to migrate the kit password. What do you think?